force-app/main/default/classes/JobApplicationTriggerHandler.cls

```diff
@@ -1,5 +1,17 @@
+/**
+ * @description       : Operations on the JobApplication Object
+ * @author            : @William, @Tyler @Fritz
```

Comment:
In other Java Docs you have created you have switched between github profile name and real names, suggest using only one to avoid confusion on behalf of readers or reviewers in the future.